pkgs/tools/misc/ipxe/default.nix

```
@@ -20,14 +18,16 @@ let
 in
 
 stdenv.mkDerivation {
```

Also needs `rec` so that `version` can be used in `fetchFromGitHub`

```suggestion
stdenv.mkDerivation rec {
```

---

done.